libs/perun/dialogs/src/lib/anonymize-user-dialog/anonymize-user-dialog.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatTableDataSource } from '@angular/material/table';
+import { User, UsersManagerService, Vo } from '@perun-web-apps/perun/openapi';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { NotificatorService } from '@perun-web-apps/perun/services';
+import { TranslateService } from '@ngx-translate/core';
+
+export interface AnonymizeUserDialogComponentData {
+  users: User[];
```

Comment:
Please pass a single `User` or make the `anonymizeUser` function handle multiple users correctly.